### DISCUSSION TITLE
post/kaspersky-hips/

### DISCUSSION THREAD

**Discussion by giscus** [lyc8503/lyc8503#16] [Announcements]
# post/kaspersky-hips/

之前写过一篇文章讲如何使用 TPM 进行 SSH 认证, 告别了用着 id_rsa/id_ed25519 裸奔的年代, 解决了我的一个心头大患. 不过电脑中除了 SSH 私钥, 还有一些其他较为关键/隐私的文件夹需要保护, 比如浏览器 User Data, XShell 配置文件夹, 开始菜单启动项等. 既然 Windows (因为一堆历史包袱) 不做应用隔离, 我就自己来做.

https://blog.lyc8503.site/post/kaspersky-hips/

**Comment by lyc8503** [comment]
Btw, 本文只是以卡巴斯基为例, 还有一些更高级的 HIPS 例如 Comodo, 其他杀软比如 ESET 应该也能实现类似效果. 我用卡巴斯基主要是也要用到它本体的杀毒功能, 你完全可以自行选择你喜欢的杀软配置 HIPS.
